_pages/en_US/finalizing-setup.txt

```diff
@@ -131,17 +131,18 @@ If, before following this guide, you already had an EmuNAND setup and would like
   + This process will take some time
   + If you get an error, make sure that you have at least 1.3GB of free space on your SD card
 1. Press (A) to continue
-1. Press (B) to return to the main menu
-1. Select "Exit"
-1. Press (A) to relock write permissions if prompted
```

Comment:
Please double-check where this line can be placed. It still prompts to relock write permissions when exiting to the main screen as the perms were unlocked while setting up Luma3DS to CTRNAND; however, this relocking step is removed here and is not referenced again.

An easy way to do this is to actually follow along the process yourself, documenting every single step you take.